h050706/Program.cs

```diff
@@ -0,0 +1,293 @@
+﻿using h050706.EnumPokemon;
```

Comment:
Mivel miden Static method ebben az osztályban, nagy részük akár lehetett volna extension method is.